src/helpers/storage.js

```diff
@@ -29,4 +29,18 @@ export const deleteTaskStorage = (id) => {
   saveDataInStorageForKey(STORAGE_KEY, newTaskList);
 };
 
+export const addCheckedTaskToStorage = (id) => {
```

Comment:
You do not need 2 separate function to editing checked/unchecked task. This is not efficient as when DB will grow, when we will add more fields, taking this approach you would need to have separate handler function for each field and a lot of duplicate code. Put all of this in one updateTaskInStorage() function

Reply:
changed in storage.js